complex/ft_config.c

```diff
@@ -669,7 +669,7 @@ void fts_cur_info(struct ft_series *series, struct ft_info *info)
 	info->class_function = set->class_function[series->cur_func];
 	info->op = set->op[series->cur_op];
 	info->test_flags = set->test_flags;
-	info->caps = set->caps[series->cur_caps];
+	info->caps = set->caps[series->cur_caps] | FT_CAP_MSG;
```

Comment:
I think the test configs may need to handle this if RMA and atomic need MSG capabilities.  But we'll eventually want to be able to test EPs that will not be configured for message operations.  We don't want to always use sockets, since that will impact any performance measurements that we may want to collect.  Maybe the framework can be adaptive and use in or out of band communication as needed?